Libraries/LibWeb/Streams/ReadableByteStreamController.h

```diff
@@ -172,7 +194,7 @@ class ReadableByteStreamController : public Bindings::PlatformObject {
 
     // https://streams.spec.whatwg.org/#readablebytestreamcontroller-pendingpullintos
     // A list of pull-into descriptors
-    SinglyLinkedList<PullIntoDescriptor> m_pending_pull_intos;
+    SinglyLinkedList<GC::Ref<PullIntoDescriptor>> m_pending_pull_intos;
```

Comment:
Would an `IntrusiveList` make even more sense here? Can a `PullIntoDescriptor` be in more than one list at once?